lib/browser/api/dialog.js

```diff
@@ -185,10 +197,10 @@ module.exports = {
       showsTagField = true
     }
 
-    const wrappedCallback = typeof callback === 'function' ? function (success, result) {
-      return callback(success ? result : void 0)
+    const wrappedCallback = typeof callback === 'function' ? function (success, result, bookmarkData) {
+      return success ? callback(result, bookmarkData) : callback(void 0) // eslint-disable-line
```

Comment:
There is absolutely no reasons to use `void 0`. Just call the callback without arguments.
(Which doesn't make much sense either, but since the API already looks like that it's ok.)

Reply:
Cool, I'm happy with that. I originally used `void 0` since that was what was there before my changes.

Reply:
> I originally used `void 0` since that was what was there before my changes.

Sure, I totally understand it. But let's leave the code better than we found it.

Reply:
👍